docs/migration/multi-project-to-multi-project.md

```diff
@@ -273,6 +273,14 @@ public static class MauiProgram
 }
 ```
 
+> [!Note]
+> For Xamarin.UWP projects, the `App` reference in `builder.UseMauiApp<App>()` is found in the MainPage.xaml.cs file.
+
+If there are platform specific services that need to be migrated over to .NET MAUI, utilize `builder.Services.AddTransient<IServiceCollection, Type>();` to add a transient service of the type specified in serviceType to the specified IServiceCollection. See, <xref:Microsoft.Extensions.DependencyInjection.ServiceCollectionServiceExtensions.AddTransient> for more information on the ServiceCollectionServiceExtensions.AddTransient method.
```

Comment:
I'm trying to reference [this link here](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.servicecollectionserviceextensions.addtransient?view=dotnet-plat-ext-8.0#microsoft-extensions-dependencyinjection-servicecollectionserviceextensions-addtransient(microsoft-extensions-dependencyinjection-iservicecollection-system-type)), trying to figure out why it isn't working